extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -127,12 +129,16 @@ define('ext.wikia.adEngine.video.googleIma', [
 						self.adDisplayContainer.initialize();
 						self.adsManager.init(width, height, google.ima.ViewMode.NORMAL);
 						self.adsManager.start();
+						log('Video play: stared', log.levels.debug, logGroup);
 					};
 
 				if (this.isAdsManagerLoaded) {
 					callback();
-				} else {
+				} else if (!browserDetect.isMobile()) { // ADEN-4275 quick fix
+					log(['Video play: waiting for full load of adsManager'], log.levels.debug, logGroup);
 					this.adsLoader.addEventListener(google.ima.AdsManagerLoadedEvent.Type.ADS_MANAGER_LOADED, callback, false);
+				} else {
+					log(['Video play: trigger video play action is ignored'], log.levels.warning, logGroup);
```

Comment:
Right now I don't know how browser on iPads behaves, but I guess same as for mobile devices (throw errors) what means we want to ignore it